### PR TITLE
Add Honorable Members to our community. RIP Jean-Thierry

### PR DESCRIPTION
**Pull request by MikeAlhayek**

### CODE REVIEW

File: src/docs/community/jean-thierry/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# In honor of Jean-Thierry Kéchichian
+
+![image](https://github.com/OrchardCMS/OrchardCore/assets/24724371/f73763e8-874d-443e-868a-ceaa104951c7)
```
Comment (Piedone) [inline src/docs/community/jean-thierry/README.md:3]: A bit better would be to add the image to the docs project.

File: src/docs/community/jean-thierry/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+It is with sadness that we grieve the loss of [Jean Thierry Kéchichian](https://github.com/jtkech) on the morning of December 24, 2023. His incredible kindness, unwavering assistance and vast knowledge have left an impression on our community. Jean-Thierry was a guiding light to provide detailed answers and educate those in search of guidance. His dedication to the OrchardCore project was unmatched. His contributions will forever resonate within the spaces he helped shape.
```
Comment (Piedone) [inline src/docs/community/jean-thierry/README.md:5]:
```suggestion
It is with sadness that we grieve the loss of [Jean Thierry Kéchichian](https://github.com/jtkech) on the morning of December 24, 2023. His incredible kindness, unwavering assistance and vast knowledge have left an impression on our community. Jean-Thierry was a guiding light to provide detailed answers and educate those in search of guidance. His dedication to the Orchard Core project was unmatched. His contributions will forever resonate within the spaces he helped shape.
```

File: src/docs/community/jean-thierry/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+Jean-Thierry never turned anyone away in need; he always extended a helping hand with grace. His legacy can be found not only in the lines of code he wrote, or the detailed comments he blessed us with. But, in the countless lives he touched through his generosity and warmth. The void left by his absence is immeasurable. We will deeply miss him in our community.
+
+As we navigate this time let us remember and celebrate the person that Jean Thierry was – a friend, mentor and true pillar of our community. His spirit will continue to live on through the projects he championed. The knowledge he generously shared.
+
+Rest peacefully Jean Thierry Kéchichian. Your impact will. Your memory will forever inspire us.
```
Comment (agriffard) [inline src/docs/community/jean-thierry/README.md:11]: Is there a a missing verb in the sentence `Your impact will` or is there an implied one somewhere I am missing?

**Review by Piedone — CHANGES_REQUESTED** [review]
Very nice, and very sad.

File: src/docs/community/jean-thierry/README.md (hunk 3 above)
Comment (Piedone) [reply src/docs/community/jean-thierry/README.md:11]: A truncated sentence I guess: "Your impact will [rest peacefully]".

Comment (MikeAlhayek) [reply src/docs/community/jean-thierry/README.md]: fixed.

File: src/docs/community/jean-thierry/README.md (hunk 1 above)
Comment (MikeAlhayek) [reply src/docs/community/jean-thierry/README.md]: yes. I added the images to the project.

**Comment by MikeAlhayek** [comment]
@agriffard  @Piedone is the new menu item okay?

**Comment by Piedone** [comment]
Yeah, fine by me.

**Pull request merged**